packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -6210,6 +6210,21 @@ void main() {
           );
         }, throwsAssertionError);
       });
+
+      // Regression test for https://github.com/flutter/flutter/issues/174784.
+      testWidgets('InputDecorator error widget text style defaults to errorStyle', (
+        WidgetTester tester,
+      ) async {
+        await tester.pumpWidget(
+          buildInputDecorator(decoration: const InputDecoration(error: Text(errorText))),
+        );
+
+        expect(findError(), findsOneWidget);
```

Comment:
Reading the documentation of `findsOneWidget`, it is written that

> This is equivalent to the preferred [findsOne] method.

Should we be using `findsOne` instead of `findsOneWidget`?

Reply:
> Should we be using findsOne instead of findsOneWidget?

Interesting question!
Searching through the codebase:
- findsOne: 8218
- findsOneWidget: 7610

In input_decorator_test.dart file, there are no occurences of `findsOne`, only `findsOneWidget`.
The reason is probably because `findsOne` was introduced somewhat recently, see https://github.com/flutter/flutter/pull/127137.
I would say using findsOneWidget when we are looking for a Widget is ok, but maybe @justinmc has some thinking about this.